src/couch_epi_data.erl

```diff
@@ -88,7 +88,8 @@ handle_call(stop, _From, State) ->
 handle_call(_Request, _From, State) ->
     {reply, ok, State}.
 
-handle_cast(init, #state{pending = Pending} = State) ->
+handle_cast(init, #state{pending = Pending, handle = Handle} = State) ->
+    couch_epi_module_keeper:maybe_start_keeper(couch_epi_data_gen, Handle),
```

Comment:
What? The entire point of that API was to prevent exactly this situation. Cause what happens if we want to run mem3 in a deployment without chttpd at all?